### DISCUSSION TITLE
Add search lights and discharge coils as hard point options

### DISCUSSION THREAD

**Discussion by Hinny** [FakeFishGames/Barotrauma#7216] [Ideas and suggestions]
I think that it would be cool if search lights, and electrical discharge coils could be cheap budget options for weapon hardpoints. This will allow for some extra in game customisation.

The search lights would be set up as normal (periscope and no ammo loader).

Discharge coils could be linked to the fire button on the periscope (or periscope is replaced by a button to be fired "blindly" if you think that is more thematic).

**Comment by mLuby** [comment]
Great idea. Don't forget about the humble Depth Charge Loader/Tube! For consistency, I'd say Discharge Coils and Depth Charge Tubes _should_ have the periscope linked to a camera so you can see what you're zapping or dropping.

Also a robotic arm would be nice to mount: https://github.com/Regalis11/Barotrauma/discussions/7011

**Reply by Hinny** [reply]
Yeah, depth charges would also fit nicely into this concept (and robotic arms as well I would imagine).

I agree that it is probably best if the  Discharge coils / depth charge tubes are periscope linked for maximum precision / satisfaction.
